deepface/modules/preprocessing.py

```diff
@@ -129,3 +143,58 @@ def normalize_input(img: np.ndarray, normalization: str = "base") -> np.ndarray:
         raise ValueError(f"unimplemented normalization type - {normalization}")
 
     return img
+
+
+def extract_frames_with_faces(video_bytes):
+  """Extracts frames with faces from a video as base64 bytes.
+
+  Args:
+    video_bytes: The video bytes.
+
+  Returns:
+    A list of base64-encoded frames with faces.
+  """
+  video_bytes = base64.b64decode(video_bytes)
+
+  # Create a file-like object to write the video to.
+  hash = str(random.getrandbits(128))
+  with io.open("./"+ hash + ".mp4", "wb") as f:
```

Comment:
you may consider to set a variable like target_file and use same variable while deleting the file

Reply:
cv2.VideoCapture cannot load the bytes directly, I tried using io to pretend the video bytes like streaming video but failed. Thus, I have to store at disk as temp file to load it. Do you have better suggestion? thanks

Reply:
i mean 

target_file = "./"+ hash + ".mp4"
with io.open(target_file , "wb") as f:
...
os.remove(target_file)

Reply:
sure, will be update in the next version, thanks